Objects/typeobject.c

```diff
@@ -3164,6 +3164,25 @@ _PyType_LookupId(PyTypeObject *type, struct _Py_Identifier *name)
     return _PyType_Lookup(type, oname);
 }
 
+/* Check if the "readied" PyUnicode name
+   is a double-underscore special name. */
+static int
+is_dunder_name(PyObject *name)
+{
+    Py_ssize_t length = PyUnicode_GET_LENGTH(name);
+    int kind = PyUnicode_KIND(name);
+    /* Special names contain at least "__x__" and are always ASCII. */
+    if (length > 4 && kind == PyUnicode_1BYTE_KIND) {
+        Py_UCS1 *characters = PyUnicode_1BYTE_DATA(name);
+        /* Allow the C compiler to read 2 bytes instead of two times 1 byte. */
```

Comment:
I'm not sure why this comment is here.   What would cause it to reader two bytes at a time?

There GCC-8 disassembly shows fetch and compares one byte at a time:

```
L4922:
    cmpb    $95, (%rax)
    jne L4918
    cmpb    $95, 1(%rax)
    je  L4940
    ...

L4940:
    cmpb    $95, -2(%rax,%rcx)
    jne L4918
    cmpb    $95, -1(%rax,%rcx)
    jne L4918
    movq    %rdi, %rsi
    movq    %rbx, %rdi
    call    _update_slot
    movl    %eax, %ebp
```

Reply:
This has already been changed.